src/Shared/IdentityClient/Config/ConfigServicesIdentityService.cs

```diff
@@ -11,29 +15,29 @@ public static IServiceCollection IdentityApplicationBuilderExtensions(this IServ
             services
                 .AddAuthorization(options =>
                 {
-                    options.AddPolicy("admin", policy => policy.RequireClaim("Admin"));
+                    options.AddPolicy(UserPolicy.User, policy => policy.RequireAssertion(context => context.User.IsInRole("User")));

```

Comment:
Are this policies required? I'm pretty sure, that Authorize attribute by default uses roles

Reply:
no, they are not required I have removed this